From 6f6ef6340cad65912a1ae2b40a48a037513e1e0f Mon Sep 17 00:00:00 2001
From: Joone Hur <joone.hur@intel.com>
Date: Sat, 10 Nov 2018 11:41:00 -0800
Subject: [PATCH 08/14] Build fix

| In file included from ../../ui/ozone/demo/renderer_factory.cc:5:0:
| ../../ui/ozone/demo/renderer_factory.h:21:16: error: 'unique_ptr' in namespace 'std' does not name a template type
|    virtual std::unique_ptr<Renderer> CreateRenderer(
|                 ^~~~~~~~~~
---
 ui/ozone/demo/renderer_factory.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/ui/ozone/demo/renderer_factory.h b/ui/ozone/demo/renderer_factory.h
index a39a3b416c0c..e2fc9a6401bc 100644
--- a/ui/ozone/demo/renderer_factory.h
+++ b/ui/ozone/demo/renderer_factory.h
@@ -5,6 +5,8 @@
 #ifndef UI_OZONE_DEMO_RENDERER_FACTORY_H_
 #define UI_OZONE_DEMO_RENDERER_FACTORY_H_
 
+#include <memory>
+
 #include "ui/gfx/geometry/size.h"
 #include "ui/gfx/native_widget_types.h"
 
-- 
2.20.1

